bin/varnishd/cache/cache.h

```diff
@@ -390,7 +390,8 @@ struct busyobj {
 	 * All fields from retries and down are zeroed when the busyobj
 	 * is recycled.
 	 */
-	int			retries;
+	unsigned		retries;
+	enum req_body_state_e	initial_req_body_status;
```

Comment:
This is an ABI breakage, unclear whether it belongs to the VRT or strict ABI, but most likely strict and OK: #3355.